doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+is a library for writing Plutus contracts. It provides the Contract Monad interface 
+for writing the off-chain part of a Plutus application that is to be interpreted 
+by an emulator or a plutus application backend (PAB). 
+
++--------------+---------------------------------------------------------------+
+| Contract monad                                                               |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core            |
++--------------+---------------------------------------------------------------+
+| State        | Maintained but not extended. Unstable interface.              | 
++--------------+---------------------------------------------------------------+
+
+8. Contract monad emulator
+-----------------------------------------------------------
+
+`Contract monad emulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides an environment for emulating the blockchain. The environment 
+provides a way for writing traces for the contract which are sequences of actions 
+by simulated wallets that use the contract. The component is highly dependent on the 
+Contract Monad. However, there are plans to decouple it from the Contract Monad 
+so that it can be used with other libraries.
+
++--------------+---------------------------------------------------------------+
+| Contract monad emulator                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core,           |
+|              | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | Maintained and will be extended. Unstable interface.          | 
++--------------+---------------------------------------------------------------+
+
+9. Plutus contract state machine
+-----------------------------------------------------------
+
+`Plutus contract state machine <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides a high-level model for writing a Plutus application based 
+on the State Machine formalism. Even though it is inefficient, it can be used to 
```

Comment:
By plutus application do we mean smart contract?

Reply:
I believe so. Do the terms "Plutus application" and "smart contract" always mean the same thing when we talk about the Tools? If so, should we favor the term "smart contract" generally

Reply:
I like "plutus applications" (which includes on-chain + off-chain code using Plutus in Cardano) better than "smart contracts" because it's more precise. Downside is that it's not as notorious as the term "smart contract".

Reply:
Given all these comments, I think I should use both terms and make it clear that they are referring to the same thing.